cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPullStrategy.java

```diff
@@ -269,14 +269,16 @@ private int processOneChangesBatch(ChangesResultWrapper changeFeeds)
 
             if (this.cancel) { break; }
 
-            List<Callable<DocumentRevsList>> tasks = createTasks(batch, missingRevisions);
+            Callable<Iterable<DocumentRevsList>> tasks = createTask(batch, missingRevisions);
```

Comment:
Why no more multiple tasks here? Isn't there still an advantage to operating in parallel for the case where `_bulk_get` isn't supported?